src/components/Swap/AdvancedSwapDetails.tsx

```diff
@@ -61,11 +70,32 @@ export const TradeSummary: React.FC<TradeSummaryProps> = ({
     allowedSlippage,
   );
   const classes = useStyles();
-  const showRoute = Boolean(trade && trade.route.path.length > 2);
   const tradeAmount = isExactIn ? trade.outputAmount : trade.inputAmount;
 
   return (
-    <>
+    <Box mt={1.5}>
+      {openSettingsModal && (
+        <SettingsModal
+          open={openSettingsModal}
+          onClose={() => setOpenSettingsModal(false)}
+        />
+      )}
+      <Box className={classes.summaryRow}>
+        <Box display='flex' alignItems='center'>
+          <Typography variant='body2'>Slippage:</Typography>
+          <QuestionHelper text='Your transaction will revert if the price changes unfavorably by more than this percentage.' />
```

Comment:
Should be in messages

Reply:
Put the messages for question helpers to JSON and pushed a commit